### PR TITLE
added initial layer structure

### PR DESCRIPTION
**Pull request by princekumarLnG**
added four layers in the solution of PSPCL.

### CODE REVIEW

File: Pspcl/Pspcl.API/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.API
+{
+    public class Class1
+    {
+
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.API/Class1.cs]: Why this class is added.?

File: Pspcl/Pspcl.DBConnect/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.DBConnect
+{
+    public class Class1
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.DBConnect/Class1.cs]: Why this class is added.?

File: Pspcl/Pspcl.Models/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.Models
+{
+    public class Class1
+    {
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Models/Class1.cs]: Why this class is added.?

File: Pspcl/Pspcl.Services/Class1.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.Services
+{
+    public class Class1
+    {
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Services/Class1.cs]: Why this class is added.?

File: Pspcl/Pspcl.API/Class1.cs (hunk 1 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.API/Class1.cs]: by default it was added. I'll remove it now.

File: Pspcl/Pspcl.DBConnect/Class1.cs (hunk 2 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.DBConnect/Class1.cs]: by default it was added. I'll remove it now.

File: Pspcl/Pspcl.Models/Class1.cs (hunk 3 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.Models/Class1.cs]: by default it was added. I'll remove it now.

File: Pspcl/Pspcl.Services/Class1.cs (hunk 4 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.Services/Class1.cs]: by default it was added. I'll remove it now.

**Comment by princekumarLnG** [comment]
i've removed those class1.cs files from all project which was added by default.

**Comment by princekumarLnG** [comment]
i need to make a fresh layered structure, so i'm closing this PR.

**Pull request closed without merging**